org.civicrm.reqangular/test/services/format-currency.spec.js

```diff
@@ -30,21 +30,22 @@ define([
     var testMap = [
       // [thousandSeparator, decimalSeparator, inputAmount, outputAmount]
       // Common cases
+      [',', '*', 0.0, '0'],
       [',', '*', 12345675, '12,345,675'],
       [',', '*', 123456.75, '123,456*75'],
       [',', '*', '123456.7.5', '12,345,675'],
       [',', '*', '12345675', '12,345,675'],
       [',', '*', '12,345,675', '12,345,675'],
       [',', '*', '00,456,75*76', '45,675*76'],
       [',', '*', '12345675*76', '12,345,675*76'],
-      [',', '*', '12,34,56*75*76', '123,456*7576'],
+      [',', '*', '12,34,56*75*76', '123,456*75'],
       [',', '*', '123 456 75*76', '12,345,675*76'],
 
       // Edge cases
       ['?', '^', 'abcd!89', '89'],
       ['?', '!', '00000!89', '0!89'],
       ["'", ',', '1234567,76', "1'234'567,76"],
-      ['*', '^', '1234567^76', '1*234*567^76'],
```

Comment:
Why has this been changed?

Reply:
Seems I changed the value during fixing tests, but no worries its still valid.